pyproject.toml

```diff
@@ -58,7 +58,8 @@ include = [
   "pyccel/stdlib/**/*.c",
   "pyccel/stdlib/**/*.f90",
   "pyccel/extensions/STC/include",
-  "pyccel/extensions/gFTL/include/v2"
+  "pyccel/extensions/gFTL/include/v2",
+  "pyccel/extensions",
```

Comment:
This change should not be done. It will lead to STC docs etc being included

Reply:
Fixed : c23fe61